packages/@coorpacademy-components/src/molecule/go1-base-banner/test/fixtures/default.js

```diff
@@ -0,0 +1,11 @@
+export default {
+  props: {
+    title: 'Learning Priorities, <br/> Focus on What Matters Most',
+    subTitle:
+      'Spotlight essential skills and direct learners to the most impactful training, <br/> optimizing their growth and enhancing your learning strategy.',
+    buttonLabel: 'Try Learning priorities',
+    onClick: () => console.log('click'),
+    image:
+      'https://s3-alpha-sig.figma.com/img/4c6f/87c0/99896b9146631494f8dea4f523e3cdff?Expires=1731888000&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=HvkcSCRSjz7RvLzB8D0HWNhuk7NUXFZSwLjBKmXxf5NPt-UCVf3wY9VUq~-bESu59PMJJVOcsdE-2h60AY65uswm-yk1yq2fd2dGvkeSXopK0YpqH-XLzdWo8FqHXVwSvB5RQlmA1KDHj1ERZjZJNijo2qXTE3eF71ksXxDm1g3c3XZgdD9TFfy8p9lLqKSBRpvway8bv7QNEcvuN-Q63-OR3EHHiftrZ-pOn0m2GFbuqIv3el-k3Zo60ZNINVeMjYkwN4~BG6PifpZdPWG9D3UenP12XNhPVCA3UJ7H1~ZTzu7q8jRG~h6TywD8S7sgowbtoQYoL5DE21bOxg0Ztg__'
```

Comment:
🙈

Reply:
Actually, the image I sent you to upload on s3 is unusable as it has a transparent background (i must have messed up the export 😆), and found out we can use correct image with this link. Can we keep it this way? Or we have to upload it to our s3 to use?

Reply:
we have to upload it to s3, s3-alpha-sig.figma.com we don't have control over it.
(also, please add the new image to the PR, and send it to me so I reupload it)